PactNet/Mocks/MockHttpService/MockProviderService.cs

```diff
@@ -125,32 +127,48 @@ public void VerifyInteractions()
             Async.RunSync(() => _adminHttpClient.SendAdminHttpRequest(HttpVerb.Get, $"{Constants.InteractionsVerificationPath}?example_description={testContext}"));
         }
 
-        public void SendAdminHttpRequest(HttpVerb method, string path)
+        public void SendAdminHttpRequest(HttpVerb method, string path, Dictionary<string, string> headers = null)
         {
-            Async.RunSync(() => _adminHttpClient.SendAdminHttpRequest(method, path));
+            Async.RunSync(() => _adminHttpClient.SendAdminHttpRequest(method, path, headers:headers));
         }
 
         public void Start()
         {
-            StopRunningHost();
+            if (UseRemoteMockService == false)
+            {
+                StopRunningHost();
+                _host = _hostFactory(BaseUri);
+                _host.Start();
+            }
 
-            _host = _hostFactory(BaseUri);
-            _host.Start();
         }
 
         public void Stop()
         {
             ClearAllState();
-            StopRunningHost();
+            if (UseRemoteMockService == false)
+            {
+                StopRunningHost();
+            }
         }
 
         public void ClearInteractions()
         {
-            if (_host != null)
+            if (_host != null && UseRemoteMockService == false)

```

Comment:
If something is a bool it is a little bite more cleaner to not compare this to true or false. So above condition can look like:`if(_host != null && !UseRemoteMockService)`

Reply:
Fixed